README.md

```diff
@@ -34,3 +35,5 @@ Run locally at [localhost:4200](http://localhost:4200/) with `ng serve`
 - **loadingTemplate** - Override the loader shown at the Viewer.
 - **allowDownload** - Toggle display the option to download files on the Viewer.
 - **invalidFormatDisplayImage** - Option to change the displayed image when the files is of an unsupported format.
+- **displayInline** -  Show the viewer inline instead of a dialog.
+- **keepOpen** -  When inline, always keeps the viewer open.
```

Comment:
If when inline this is always open, why add as a prop?

Reply:
Because with this flag, its possible to have it collapsed, and expand inline when opening a file